### ISSUE TITLE
US cases and deaths did not update for Feb 10.

### ISSUE THREAD

**Issue by rjpagano** [CSSEGISandData/COVID-19#6580]
time_series_covid19_confirmed_US.csv and 
time_series_covid19_deaths_US.csv 
say they were updated "11 hours ago," but have not in fact been updated. 
The most recent date showing is Feb 9. 
The data set should include Feb 10.

**Comment by dlluskin** [comment]
Looks to me like US cases and deaths did in fact update. But not global.

**Comment by parnaudo** [comment]
The [timeseries data ](https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_time_series/time_series_covid19_confirmed_US.csv) are still lagged a day, though you are correct that the [daily case counts ](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_daily_reports_us/02-10-2023.csv) seem to be updated

**Comment by dlluskin** [comment]
To be properly updated, the latest date in the raw file should be 2/10/23.  The two US files are updated. The two global files are not.

**Comment by rjpagano** [comment]
Strange.  We are seeing different things.  When I download
time_series_covid19_confirmed_US.csv and
time_series_covid19_deaths_US.csv
(both automatically updated about 16 hours ago)
the latest date in those two files is 2/10/23.

I have been pulling these data files daily for 843 consecutive days so that I can use the data to update a Covid website that I maintain.  This morning was the first time that my script threw an error after downloading and processing both the US and global time series data.

**Comment by dlluskin** [comment]
I and rjpagano are saying the same thing

**Comment by parnaudo** [comment]
yea weird, my job failed for time_series_covid19_confirmed_US.csv  but looks to be in there now. Strange that it doesn't look like anything updated according to git. Ghost in the machine! 👻💻 Looks like they can close this issue

**Comment by dlluskin** [comment]
No! They still need to look at GLOBAL cases and deaths which DID NOT UPDATE
